AUTHORS

```diff
@@ -7,14 +7,14 @@ Sage Weil <sage@redhat.com>
 Component Technical Leads
 -------------------------
 
-Core, RADOS   - Samuel Just <sam.just@inktank.com>
-RBD           - Josh Durgin <josh.durgin@inktank.com>
-RBD (kernel)  - Ilya Dryomov <ilya.dryomov@inktank.com>
-RGW           - Yehuda Sadeh <yehuda@inktank.com>
-CephFS        - Greg Farnum <greg@inktank.com>
+Core, RADOS   - Samuel Just <sjust@redhat.com>
+RBD           - Josh Durgin <jdurgin@redhat.com>
+RBD (kernel)  - Ilya Dryomov <idryomov@redhat.com>
+RGW           - Yehuda Sadeh <yehuda@redhat.com>
+CephFS        - Greg Farnum <gfarnum@redhat.com>
```

Comment:
@yehudasa can we take this chance to update Yan, Zheng's mail to "zyan@redhat.com"?